packages/shipjs/src/step/setup/CI/addGitHubActions.js

```diff
@@ -176,7 +177,6 @@ jobs:
           GITHUB_TOKEN: \${{ secrets.GITHUB_TOKEN }}
```

Comment:
Not related to this PR, but shouldn't this be `secrets.GH_TOKEN` ?

Reply:
No, because this logic creates new successful message in an issue and with `GH_TOKEN` it will be published from user account, I think will be better if it be github bot and `GITHUB_TOKEN`

Reply:
ahh right. thanks for the explanation!